### PR TITLE
[irods/irods#6559] Add instructions for installing irods-icommands (main)

### PR DESCRIPTION
**Pull request by alanking**
Useful for users who do not wish to build their own iCommands package
but just want to install the released packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This repository hosts the client iCommands, the default command line interface to iRODS.
 
+The client iCommands are released as a single package called `irods-icommands` in the packages.irods.org repository: https://packages.irods.org/
```
Comment (trel) [inline README.md:5]: "in the packages.irods.org repository" -> "in the repository at"

Comment (korydraughn) [inline README.md:5]: We mention `packages.irods.org` twice. Consider doing one of the following:
- Change `in the packages.irods.org` to `in the iRODS packages`
- Make `packages.irods.org` into a link and remove `: https://packages.irods.org/`

Comment (korydraughn) [reply README.md:5]: @trel already provided a suggestion.

Feel free to ignore this.

Comment (alanking) [reply README.md]: Done

Comment (alanking) [reply README.md]: I went with the other suggestion. Thanks

**Comment by trel** [comment]
squash em

**Pull request merged**